src/index.html

```diff
@@ -22,6 +22,414 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
use empty lines for more readability(ain all the cases)
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules
<img width="1061" alt="Знімок екрана 2024-08-13 о 17 32 59" src="https://github.com/user-attachments/assets/db17b145-6ca8-4a63-9a4d-aedc2dee1252">

```suggestion
          </li>
          
          <li class="nav__item">
```